mem0/client/main.py

```diff
@@ -245,6 +245,17 @@ def users(self):
         capture_client_event("client.users", self)
         return response.json()
 
+    @api_error_handler
+    def delete_users(self) -> Dict[str, Any]:
+        """Delete all users, agents, or sessions."""
```

Comment:
Looks good to me. But can you see the function description? It says delete all users/agents/sessions, but the function only deletes the users.

Reply:
Thanks, It actually deletes all the users or sessions or agents. I used the name from the above function where we are getting all the users.

So I'm confused now what to do :)

Reply:
@deshraj can help us here